test/mongoLocker.js

```diff
@@ -4,48 +4,31 @@ const assert = require('assert');
 const sinon = require('sinon');
 const { it, describe, beforeEach } = require('mocha');
 const { times, pick } = require('lodash');
-const getMongoLocker = require('../lib/getMongoLocker');
+const getMongoLocker = require('../lib/getMongoLocker').default;
+const mongoLocker = require('../lib/mongoLocker').default;
 const getMongoClient = require('./getMongoClient');
 
-describe('getMongoLocker', () => {
-
-    let mongoClient;
-    const collectionName = 'locks';
-    let collection;
-
-    beforeEach(async () => {
-        mongoClient = await getMongoClient();
-        collection = mongoClient.db()
-            .collection(collectionName);
-        try {
-            await collection.drop();
-        } catch (e) {
-        } // eslint-disable-line no-empty
-    });
+describe('mongoLocker', () => {
 
     it('should acquire and release a lock', async () => {
 
         let i = 0;
         const callback = sinon.spy(() => ++i);
 
-        const locker = await getMongoLocker(collection);
-
-        const result1 = await locker('AAA', callback);
+        const result1 = await mongoLocker('AAA', callback);
```

Comment:
I think it would be better to copy-paste some of these tests and have them for both - the mongoLocker and getMongoLocker to be 100% sure its backward compatible
(or just extend the tests to use both and then check if both works the same and correctly)